src/main/java/com/google/devtools/build/lib/remote/disk/DiskCacheClient.java

```diff
@@ -143,22 +145,31 @@ public void close() {}
     return Futures.immediateFuture(ImmutableSet.copyOf(digests));
   }
 
+  protected Path toPathNoSplit(String key) {
+    return root.getChild(key);
+  }
+
   protected Path toPath(String key, boolean actionResult) {
-    return root.getChild(getDiskKey(key, actionResult));
+    String cacheFolder = actionResult ? "ac" : "cas";
```

Comment:
Should use `AC_DIRECTORY` and `CAS_DIRECTORY` respectively.

Reply:
Done